src/domains/Restaurants/ManageRestaurants.tsx

```diff
@@ -27,38 +33,101 @@ export const ManageRestaurants = () => {
       });
       setLastSubmittedRestaurant(restaurant.name);
       setValue("name", "");
+      setValue("tags", []);
     } catch (e) {
       console.error(e);
     }
   };
 
+  const validateRestaurentInput = () => {
+    return restaurant.name;
+  };
+
   return (
     <MainContainer>
-      Add a restaurant
-      <br />
-      <br />
-      {/**This form disables autocomplete on the text input*/}
-      <form autoComplete="off">
-        <input
-          autoComplete="false"
-          name="hidden"
-          type="text"
-          style={{ display: "none" }}
-        />
-        <FormsTextInput
-          control={control}
-          label="Name of restaurant"
-          name="name"
-        />
-      </form>
-      <br />
-      <PrimaryButton onClick={onSubmit}>Submit</PrimaryButton>
+      <Stack direction="row" spacing={2}>
+        <Box>
+          {/**This form disables autocomplete on the text input*/}
+          <form autoComplete="off">
+            <input
+              autoComplete="false"
+              name="hidden"
+              type="text"
+              style={{ display: "none" }}
+            />
+            <FormsTextInput
+              control={control}
+              description={"What kind of food do they have?"}
+              descriptionPlacement="top"
+              label="Name of restaurant"
+              name="name"
+            />
+          </form>
+          <br />
+        </Box>
+        <Box>
+          {/**This form disables autocomplete on the text input*/}
+          <FormsMultiSelect<string>
+            control={control}
+            label={"Tags"}
+            name={"tags"}
+            description={"What kind of food do they have?"}
+            descriptionPlacement="top"
+            options={["Burgers", "Pizza", "Greek"]}
+            defaultValue={["Burgers"]}
+            getOptionLabel={(option: string) => option}
+          />
+          <br />
+        </Box>
+      </Stack>
+      <PrimaryButton disabled={!validateRestaurentInput()} onClick={onSubmit}>
+        Submit
+      </PrimaryButton>
       {lastSubmittedRestaurant && (
         <>
           <br />
-          Got it - added {lastSubmittedRestaurant}
+          {lastSubmittedRestaurant} has been added to your list!
         </>
       )}
+
```

Comment:
This should be moved out to it's own component. 

To see how, just look at any existing component and copy that pattern